draft-ietf-ppm-dap.md

```diff
@@ -868,15 +869,14 @@ struct {
       to link a report back to the Client that generated it.
 
 * `public_share` is the public share output by the VDAF sharding algorithm. Note
-  that the public share might be empty, depending on the VDAF. For example,
-  Prio3 has an empty public share, but Poplar1 does not. See {{!VDAF}}.
```

Comment:
Reviewer note: As of VDAF-04, Prio3 has a non-empty public share (the joint randomness parts).